relay/config.example.yml

```diff
@@ -14,10 +14,15 @@ processing:
 
 # In some cases, relay might fail to find out the actual machine memory
 # therefore it makes the healthcheck fail and events can't be submitted.
+# See https://github.com/getsentry/self-hosted/issues/3330
 # As a workaround, uncomment the following line:
 #
 # health:
 #  max_memory_percent: 1.0
+# spool:
+#   envelopes:
+#     path: "/your/path"
```

Comment:
Can we have an example of this path?

Reply:
It should just be a path residing on the Docker volume... or since relay doesn't have a persisted volume, probably it's best to use `/tmp/foo` directory?

Reply:
I'm quite unfamiliar with this change, so could you please bear with me and tell me, is it going to be more than 4GB?

I've seen odd situations when using `/tmp` and filling it with files bigger than 4GB size.

If that's not the case, then correct, `/tmp/foo` seems right to me.

Reply:
Who not mount a volume to `relay` for this?

Reply:
From the docs, the default size is 500MB. We should be fine, no need for another volume. https://docs.sentry.io/product/relay/options/#spooling